File: storage_sdk/src/main/java/com/walker/storage/room/model/Word.java
package com.walker.storage.room.model;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * Created by walkerzpli on 2021/7/30.
 */

@Entity(tableName = "word_table")
public class Word {

    @PrimaryKey(autoGenerate = true)
    private int id;

    @NonNull
    @ColumnInfo(name = "content")
    private String mContent;

    @ColumnInfo(name = "createTime")
    private long mCreateTime;


    public Word(@NonNull String content) {
        this.mContent = content;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public String getContent() {
        return mContent;
    }

    public long getCreateTime() {
        return mCreateTime;
    }

    public void setCreateTime(long createTime) {
        this.mCreateTime = createTime;
    }


    @Override
    public String toString() {
        return "Word{" +
                "id=" + id +
                ", mContent='" + mContent + '\'' +
                ", mCreateTime=" + mCreateTime +
                '}';
    }
}

File: storage-ktx/src/main/java/com/demo/storage_ktx/model/Word.kt
package com.demo.storage_ktx.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.demo.storage_ktx.Constants

/**
 * Created by lizhiping on 2023/6/5.
 * <p>
 * description
 */
@Entity(tableName = Constants.TABLE_WORD)
data class Word(
    @PrimaryKey val id: Int? = null,
    @ColumnInfo(name = "word") val word: String
)
